.github/workflows/report-viewer.yml

```diff
@@ -24,9 +24,9 @@ jobs:
           MAJOR=$(echo $VERSION | cut -d '.' -f 1)
           MINOR=$(echo $VERSION | cut -d '.' -f 2)
           PATCH=$(echo $VERSION | cut -d '.' -f 3)
-          sed -i "/major/s/.*/    \"major\": $MAJOR,/" report-viewer/src/version.json
-          sed -i "/minor/s/.*/    \"minor\": $MINOR,/" report-viewer/src/version.json
-          sed -i "/patch/s/.*/    \"patch\": $PATCH/" report-viewer/src/version.json
+          sed -i "3 s/.*/    \"major\": $MAJOR,/" report-viewer/src/version.json
+          sed -i "4 s/.*/    \"minor\": $MINOR,/" report-viewer/src/version.json
+          sed -i "5 s/.*/    \"patch\": $PATCH/" report-viewer/src/version.json
```

Comment:
Are these hard coded line numbers of the `version.json`? We should at least add a comment to document the dependency

Reply:
Yes. I did not find a better way of only editing the first occurance of each string. Maybe @dfuchss knows a better way?

Reply:
According to some AI model we all know, something like that should work:

```bash
json=$(cat example.json)

json=$(echo "$json" | jq --arg MAJOR "$MAJOR" --arg MINOR "$MINOR" --arg PATCH "$PATCH" '.report_viewer_version |= { "major": $MAJOR | tonumber, "minor": $MINOR | tonumber, "patch": $PATCH | tonumber }')

echo "$json" > example.json

```

Reply:
Tested it on a fork an works just like expected👍  Thank you